### PR TITLE
migrate kubeflow jobs to its own config subdir

### PR DESCRIPTION
**Pull request by krzyzacy**
simple copy paste, and added @jlewi as owner so he can approve kubeflow config changes in the future.

/area jobs
/assign @jlewi @BenTheElder

### CODE REVIEW

**Comment by jlewi** [comment]
Thank you.

Should we create an OWNERs file and add ourselves so we don't need to bother you for changes?

**Comment by fejta** [comment]
awesome

File: config/jobs/kubeflow/OWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+approvers:
+- jlewi
```
Comment (krzyzacy) [inline config/jobs/kubeflow/OWNERS]: @jlewi this is the OWNERS file, feel free to change it and add more people here.

**Comment by krzyzacy** [comment]
should be a no-op, but maybe keep an eye there :-)

/hold cancel

File: config/jobs/kubeflow/OWNERS (hunk 1 above)
Comment (jlewi) [reply config/jobs/kubeflow/OWNERS]: Thanks!
/lgtm

**Comment by BenTheElder** [comment]
Yes!!
Next up: sig-node :-)

**Comment by BenTheElder** [comment]
hey look the indentation is correct!

**Comment by krzyzacy** [comment]
I think there's still a bug in certain cases :-\

**Pull request merged**